docs/development/custom-ca-citadel/README.md

```diff
@@ -0,0 +1,103 @@
+# Using a custom intermediate CA with Citadel
+
+## Background
+
+Imagine you work for a big company that runs its own PKI. Most of this companies own its own Root CA and generate
```

Comment:
Actually, I've made a mistake here, "this companies" should be "these companies" (these is the plural form of this). Sorry!